src/company_cmd.cpp

```diff
@@ -818,6 +818,7 @@ void CompanyAdminRemove(CompanyID company_id, CompanyRemoveReason reason)
  * - bits 16..24: CompanyID
  * @param p2 various depending on CompanyCtrlAction
  * - bits 0..31: ClientID (with CCA_NEW)
+ * - bit 0: If set to 1, enforce the max_no_competitors test (with CCA_NEW_AI)
```

Comment:
Why would you ever want to not enforce this?

Reply:
There are currently some places in the code where you start an ai company disregarding the max_no_competitors, like reload ai, startai console command

Reply:
Okay but why? If it bypasses max_no_competitors, what's the point of max_no_competitors?

Reply:
The point is to limit the number of AIs that can start automatically. Manually starting AIs past the limit is okay in my view, You are in control of how you want that to happen and when. It has always been like that.